pkg/services/sqlstore/org_users.go

```diff
@@ -109,7 +109,7 @@ func (ss *SQLStore) GetOrgUsers(ctx context.Context, query *models.GetOrgUsersQu
 		whereConditions = append(whereConditions, fmt.Sprintf("%s.is_service_account = ?", ss.Dialect.Quote("user")))
 		whereParams = append(whereParams, ss.Dialect.BooleanStr(false))
 
-		if ss.Cfg.IsEnterprise && !accesscontrol.IsDisabled(ss.Cfg) && query.User != nil {
+		if ss.Cfg.IsEnterprise && !accesscontrol.IsDisabled(ss.Cfg) {
```

Comment:
:question: should we panic/error/warn before if the user is nil?

Reply:
The access control filter will deny the query if user is nil. Not sure we want to panic but a warning log could be helpful

Reply:
To double check, as far as I understand, the original PR was made to fix a crash, and we don't need it now since we have the SQL filter which will avoid a crash?